docs/how-tos/train-deploy-ml.md

```diff
@@ -235,67 +247,80 @@ We have two datasets you can use for testing, one with shapes and the other with
 
 {{% /expand%}}
 
-{{< alert title="Tip" color="tip" >}}
-For best results when training machine learning models:
+{{< gif webm_src="/how-tos/capture-images.webm" mp4_src="/how-tos/capture-images.mp4" alt="Configuring data management for a camera in the viam app" max-width="600px" class="aligncenter" >}}
+
+{{< table >}}
+{{% tablestep link="/services/data/" %}}
+**1. Enable the data management service**
 
-- Provide at least 10 images of the same object, taken from different angles, and repeat this approach for each object you want your machine to be able to identify.
-  Generally, the more different perspectives of a given object you tag, the more likely it is that your model will be able to identify it, even under differing conditions.
-- Include some images that do not contain any of the objects you wish to identify, but do not add labels to these images.
-  Unlabelled images must not comprise more than 20% of your dataset, so if you have 25 images in your dataset, at least 20 of those must be labelled.
-- If you want your machine to operate successfully in various lighting conditions, such as changing sunlight, include images of each object from varying lighting conditions.
+In the configuration pane for your configured camera component, find the **Data capture** section.
+Click **Add method**.
 
-{{< /alert >}}
+When the **Create a data management service** prompt appears, click it to add the service to your machine.
+You can leave the default data manager settings.
 
-{{< table >}}
+{{% /tablestep %}}
 {{% tablestep %}}
-{{<imgproc src="/services/ml/collect.svg" class="fill alignleft" style="width: 250px" declaredimensions=true alt="Collect data">}}
-**1. Collect images**
+**2. Capture data**
 
-Start by collecting images from your cameras and syncing it to the Viam app.
-See [Collect image data and sync it to the cloud](/how-tos/image-data/#collect-image-data-and-sync-it-to-the-cloud) for instructions.
+With the data management service configured on your machine, configure how the camera component itself captures data:
```

Comment:
```suggestion
With the data management service configured on your machine, configure how the camera component captures data:
```
I don’t think “itself” is necessary, and it’s technically incorrect I think since the data service captures the output from the camera component? Don’t need to get too into the weeds but I think “itself” emphasizes a technically incorrect point.